pkgs/tools/system/howdy/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchurl
+, dpkg
+, makeWrapper
+, python3
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "howdy";
+  version = "2.6.1";
+
+  src = fetchurl {
+    url = "https://github.com/boltgolt/howdy/releases/download/v${finalAttrs.version}/howdy_${finalAttrs.version}.deb";
+    hash = "sha256-wQQynYx/VFCLMXrH+5eC6uQQQYe6xlfuioCu7rQGyE8=";
+  };
+
+  nativeBuildInputs = [ dpkg makeWrapper python3 installShellFiles ];
+
+  installPhase = ''
+    runHook preInstall
+
+    dpkg -x $src $out
+
+    runHook postInstall
+  '';
+
+  postInstall = ''
+    wrapProgram $out/lib/security/${finalAttrs.pname}/cli.py \
+        --suffix PATH : ${lib.makeBinPath [ python3 ]}
+
+    installManPage $out/usr/share/man/man1/${finalAttrs.pname}.1.gz
+
+    installShellCompletion --bash $out/usr/share/bash-completion/completions/${finalAttrs.pname}
+  '';
+
+  fixupPhase = ''
+    runHook preFixup
+
+    mkdir $out/bin
+    mv $out/lib/security/${finalAttrs.pname}/* $out/bin/
```

Comment:
```suggestion
    mv $out/lib/security/howdy/* $out/bin/
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/254359/commits/425ff1d827ff6c907684bf3451aa3fb69215491f